pkg/processor/poddisruptionbudget/pdb.go

```diff
@@ -0,0 +1,109 @@
+package poddisruptionbudget
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/arttor/helmify/pkg/processor"
+
+	"github.com/arttor/helmify/pkg/helmify"
+	yamlformat "github.com/arttor/helmify/pkg/yaml"
+	"github.com/iancoleman/strcase"
+	policyv1 "k8s.io/api/policy/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	pdbTempSpec = `
+spec:
+  minAvailable: {{ .Values.%[1]s.minAvailable }}
+  maxUnavailable: {{ .Values.%[1]s.maxUnavailable }}
+  selector:
+%[2]s
+    {{- include "%[3]s.selectorLabels" . | nindent 6 }}`
+)
+
+var pdbGVC = schema.GroupVersionKind{
+	Group:   "policy",
+	Version: "v1",
+	Kind:    "PodDisruptionBudget",
+}
+
+// New creates processor for k8s Service resource.
+func New() helmify.Processor {
+	return &pdb{}
+}
+
+type pdb struct{}
+
+// Process k8s Service object into template. Returns false if not capable of processing given resource type.
+func (r pdb) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstructured) (bool, helmify.Template, error) {
+	if obj.GroupVersionKind() != pdbGVC {
+		return false, nil, nil
+	}
+	pdb := policyv1.PodDisruptionBudget{}
+	err := runtime.DefaultUnstructuredConverter.FromUnstructured(obj.Object, &pdb)
+	if err != nil {
+		return true, nil, fmt.Errorf("%w: unable to cast to pdb", err)
+	}
+	spec := pdb.Spec
+	values := helmify.Values{}
+
+	meta, err := processor.ProcessObjMeta(appMeta, obj)
+	if err != nil {
+		return true, nil, err
+	}
+
+	name := appMeta.TrimName(obj.GetName())
+	shortName := strings.TrimPrefix(name, "controller-manager-")
```

Comment:
`"controller-manager-"`  is mostly used in k8s operator projects, so it is not relevant for `PodDisruptionBudget`.

Reply:
Right.  In my use-case I'm adding a PDB to my kubebuilder project's kustomize configuration, so it would be relevant to me.  I'm happy to remove it anyways, however, though it would be nice to be consistent :)